File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
       
        index1=-1
        index2=0
        check = True
        for num1 in nums:
            num2_can = target - num1
            index1+=1
            for index22,num2 in enumerate(nums):
                if (num2 == num2_can) and (index1 != index22):
                    # print(index22,num2)
                    index2=index22
                    check = False
                    break
            if (check == False):
                break

        return [index1,index2]




            # if (num2_can in nums):
            #     index1=nums.index(num1)
            #     index2=nums.index(num2_can)
            #     break


                
                

File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        // int index1;
        // int index2;
        vector<int> answer(2);
        
        int num1;
        int num2;
        int len = nums.size();
        bool check=0;
        for (int i=0;i<len;i++){
            num1 = nums[i];
            num2 = target - num1;
            for (int j=i;j<len;j++){
                if ((num2==nums[j]) && (i!=j)){
                    answer[0] = i;
                    answer[1] = j;
                    check=1;
                    break;
                }
            }
            if (check){
                break;
            }
        }
        
        return answer;
    }
};